frontend/.env.example

```
@@ -1,4 +1,9 @@
-# Required for local development.
+EXPO_PUBLIC_CONVEX_URL=https://your-convex-deployment.convex.cloud
+EXPO_PUBLIC_APP_ORIGIN=https://polybuys.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_APP_ORIGIN key should go before the EXPO_PUBLIC_CONVEX_URL key

(UnorderedKey)